find.go

```diff
@@ -53,6 +53,109 @@ func (s *server) findMultihashSubtree(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (s *server) findMetadata(w http.ResponseWriter, r *http.Request) {
+	var rb []byte
+	discardBody(r)
+	if r.Method != http.MethodGet {
+		http.Error(w, "", http.StatusNotFound)
+		return
+	}
+
+	rcode, resp := s.doFindMetadata(r.Context(), r.Method, findMethodOrig, r.URL, rb)
+	if rcode != http.StatusOK {
+		http.Error(w, "", rcode)
+		return
+	}
+	httpserver.WriteJsonResponse(w, http.StatusOK, resp)
+}
+
+func (s *server) doFindMetadata(ctx context.Context, method, source string, req *url.URL, body []byte) (int, []byte) {
+	start := time.Now()
+	latencyTags := []tag.Mutator{tag.Insert(metrics.Method, method)}
+	loadTags := []tag.Mutator{tag.Insert(metrics.Method, source)}
+	defer func() {
+		_ = stats.RecordWithOptions(context.Background(),
+			stats.WithTags(latencyTags...),
+			stats.WithMeasurements(metrics.FindLatency.M(float64(time.Since(start).Milliseconds()))))
+		_ = stats.RecordWithOptions(context.Background(),
+			stats.WithTags(loadTags...),
+			stats.WithMeasurements(metrics.FindLoad.M(1)))
+	}()
```

Comment:
This is going to report latency metrics that are purposed for HTTP multihash find latency. 

We should either remove metric collection for find metadata and postpone for later or define dedicated metrics/tags.